### PR TITLE
refactor(RELEASE-920): drop images support in push-snapshot

### PR DESCRIPTION
**Pull request by johnbieren**
This commit changes the push-snapshot task to no longer use the data.images key. The information in the images key is now in the data.mapping key.

This also removes the commonTags result from push-snapshot, so the pipeline when conditions that depended on it are updated to use the mapped result from apply-mapping instead.

### CODE REVIEW

File: tasks/push-snapshot/tests/test-push-snapshot.yaml [hunk 1]
```diff
@@ -78,23 +90,27 @@ spec:
...
+                echo Error: skopeo was expected to be called 8 times. Actual calls:
                 cat $(workspaces.data.path)/mock_skopeo.txt
                 exit 1
               fi
 
-              test $(jq -r '.images[0].name' $(workspaces.data.path)/results/push-snapshot-results.json) == "comp"
+              test $(jq -r '.images[0].name' $(workspaces.data.path)/results/push-snapshot-results.json) == "comp1"
               test $(jq -r '.images[0].shasum' $(workspaces.data.path)/results/push-snapshot-results.json) == \
-                "sha256:fe05fffad63015c37ef9e0c566638bd5480a5a11c710ebf7c2b91f1a5bc4a3e2"
+                "sha256:5134be780694b03e072a1735d0eea03b6e31ae265460a0b46b50424d46686f3f"
```
Comment (theflockers) [inline tasks/push-snapshot/tests/test-push-snapshot.yaml:111]: how did you get this value? I tried getting  the `sha256sum`  of the `docker://registry.io/image1:tag1` but it did not match. I might have read the mock not properly. Can you please clarify?

Comment (johnbieren) [reply tasks/push-snapshot/tests/test-push-snapshot.yaml:111]: I think it comes from https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/push-snapshot/tests/mocks.sh#L40. TBH, I just let the test fail and saw what the value was then put it in the file

Comment (theflockers) [reply tasks/push-snapshot/tests/test-push-snapshot.yaml:111]: Found. It keeps the `\n` to calculate the hash.

**Review by theflockers — APPROVED** [review]
lgtm

**Pull request merged**